enterprise/tailnet/pgcoord.go

```diff
@@ -82,7 +84,21 @@ type pgCoord struct {
 // NewPGCoord creates a high-availability coordinator that stores state in the PostgreSQL database and
 // receives notifications of updates via the pubsub.
 func NewPGCoord(ctx context.Context, logger slog.Logger, ps pubsub.Pubsub, store database.Store) (agpl.Coordinator, error) {
-	ctx, cancel := context.WithCancel(ctx)
+	ctx, cancel := context.WithCancel(dbauthz.As(ctx, rbac.Subject{
+		ID: uuid.Nil.String(),
+		Roles: rbac.Roles([]rbac.Role{
+			{
+				Name:        "tailnetcoordinator",
+				DisplayName: "Tailnet Coordinator",
+				Site: rbac.Permissions(map[string][]rbac.Action{
+					rbac.ResourceTailnetCoordinator.Type: {rbac.WildcardSymbol},
+				}),
+				Org:  map[string][]rbac.Permission{},
+				User: []rbac.Permission{},
```

Comment:
Can this be moved to `dbauthz/dbauthz.go` in the same place where `subjectProvisionerd`, `subjectAutostart` and friends are defined? (L109+)

Reply:
I could, but I chose to put it here because it's only used in one place and here it is very clear what it is for.

Reply:
Locality wins. 👍